packages/telemetry/src/events/session.ts

```diff
@@ -0,0 +1,74 @@
+import escalade from 'escalade/sync';
+import { createRequire } from 'node:module';
+import { fileURLToPath } from 'node:url';
+
+const require = createRequire(import.meta.url);
+
+const EVENT_SESSION = 'ASTRO_CLI_SESSION_STARTED';
+
+interface EventCliSession {
+	astroVersion: string;
+	cliCommand: string;
+}
+
+interface ConfigInfo {
+	hasViteConfig: boolean;
+	hasMarkdownPlugins: boolean;
```

Comment:
instead of `hasMarkdownPlugins` as a boolean, can we report on the names of plugins used? Something like `markdownPlugins: ['remark-code-titles', 'rehype-autolink-headings']` based on https://docs.astro.build/en/reference/configuration-reference/#markdownremarkplugins. It would help us figure out which features users want added to Astro's native Markdown support.